src/modules/blockchainApplication/explore/components/BlockchainApplicationRow/BlockchainApplicationRow.js

```diff
@@ -4,13 +4,14 @@ import { usePinBlockchainApplication } from '@blockchainApplication/manage/hooks
 import DialogLink from 'src/theme/dialog/link';
 import { TertiaryButton } from 'src/theme/buttons';
 import Icon from 'src/theme/Icon';
-import TokenAmount from 'src/modules/token/fungible/components/tokenAmount';
+import FormattedNumber from 'src/modules/common/components/FormattedNumber';
+import { fromRawLsk } from '@token/fungible/utils/lsk';
 import liskLogo from '../../../../../../setup/react/assets/images/LISK.png';
 import styles from './BlockchainApplicationRow.css';
 
 const DepositAmount = ({ amount }) => (
   <span className={`deposit-amount ${styles.amount} ${grid['col-xs-3']}`}>
-    <TokenAmount val={amount} token="LSK" />
+    <FormattedNumber val={fromRawLsk(amount)} /> LSK
```

Comment:
In some places you are using TokenAmount and in others you use FormattedNumber with LSK next to it. TokenAmount is essentially FormattedNumber + token symbol.
Why do you do such a change?

Reply:
yhea so reason was because in some places , we need to only render LSK amount in-regardless of the chain we are in. but since TokenAmount now takes in token as a token object to render its values, I kind of did this.

But i guess a better solution to this would be to extend TokenAmount such that it can render LSK values without issues